package.json

```diff
@@ -13,9 +13,10 @@
     "compile": "rm -rf lib/* && babel --loose es6.modules -d lib/ src/",
     "styleguide": "kss --config kss-config.json",
     "prod": "npm run clean && webpack --config webpack.prod.config.js --progress --profile --colors && npm run styleguide",
-    "lint": "eslint . --cache --cache-location=.cache/eslint --ext .js,.jsx",
+    "lint": "eslint src/bindings/redux-form/* --cache --cache-location=.cache/eslint --ext .js,.jsx",
```

Comment:
Are we only linting the redux-form folder on purpose?

Reply:
Oh yeah, I left that in by mistake.